simpleratingbar/src/main/java/com/iarcuschin/simpleratingbar/SimpleRatingBar.java

```diff
@@ -673,6 +677,56 @@ private void setNewRatingFromTouch(float x, float y) {
       }
     }
   }
+  
+  @Override
+  protected Parcelable onSaveInstanceState() {
+      Parcelable superState = super.onSaveInstanceState();
+      SavedState savedState = new SavedState(superState);
+      savedState.rating = getRating();
+      return savedState;
+    }
+
+  @Override
+  protected void onRestoreInstanceState(Parcelable state) {
+      SavedState savedState = (SavedState) state;
+      super.onRestoreInstanceState(savedState.getSuperState());
+      setRating(savedState.rating);
```

Comment:
Nice ref link. Based on that, I did some research and the stock RatingBar follows the same pattern: after restoring the state, it fires `onRatingChanged` (`fromUser` is false).
So, let's merge this PR right away :tada: